cate/ops/correlation.py

```diff
@@ -251,8 +253,14 @@ def _pearsonr(x: xr.DataArray, y: xr.DataArray, monitor: Monitor) -> xr.Dataset:
         # deferred processing.
         # Comparing with NaN produces warnings that can be safely ignored
         default_warning_settings = np.seterr(invalid='ignore')
-        r.values[r.values < -1.0] = -1.0
-        r.values[r.values > 1.0] = 1.0
+        with monitor.child(1).observing("task 1"):
+            negativ_r = r.values < -1.0
```

Comment:
@mzuehlke 
So, we only monitor when we try to access the values? As in, processing from lines 239-247 would be deferred until then?